cmds/boot/systemboot/main.go

```diff
@@ -97,6 +102,18 @@ func runIPMICommands() {
 	} else {
 		log.Printf("Watchdog is stopped.")
 	}
+	// Try RW_VPD first
+	value, err := systembooter.Get(VpdBmcBootOrderOverride, false)
+	if err != nil {
+		// Try RO_VPD
+		value, err = systembooter.Get(VpdBmcBootOrderOverride, true)
+	}
+	if err == nil && string(value) == "1" {
```

Comment:
Could you simply do this: 
if err == nil {
bmcBootOverride = true
}
log.Printf("VPD %s is %v", VpdBmcBootOrderOverride, value)

Reply:
done.